src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/zipfilestatus/ZipFileStatusServiceTest.java

```diff
@@ -175,8 +175,7 @@ public void should_return_list_of_ZipFileStatus_for_valid_dcn_parameter_search()
 
         // then
         assertThat(zipFileStatusList.size() == 2);
-        assertThat(zipFileStatusList.get(0).fileName).isEqualTo("hello1.zip");
-        assertThat(zipFileStatusList.get(1).fileName).isEqualTo("hello2.zip");
```

Comment:
why is this removed?

Reply:
When we are searching for dcn, filename and ccdid will be null. Out of 3 depending on what user is searching for, other two will be null. This test was written before this implementation so had to modify this.

Reply:
so assert them null ?

Reply:
you are right. I have modified all changes to assert null which is not there.